CHANGELOG.md

```diff
@@ -6,19 +6,35 @@ accidentally triggering the load of a previous DB version.**
 
 ## Unreleased
 
+## 2.2.1 (2021-05-05)
+
+This maintenance release contains bugfixes since the 2.1.0 release. We
+deem it high priority for upgrading.
+
+This release adds support for Skip Scan on multinode by enabling the
+pushdown of `DISTINCT` to data nodes and also fixes a number of bugs
+in the implementation of Skip Scan.
+
 **Features**
```

Comment:
Not super-important, and more of a nit: IMO this is a "fix" or a "performance" enhancement rather than a "feature". There's no "new" functionality introduced that changes capabilities. There's a lot of stuff that we can optimize in queries, especially when it comes to pushing things down to data nodes. I don't think all such enhancements are features.

Wondering if we should start adding a section for "Performance enhancement" or similar. Fine to do this for later too.

Reply:
Even though it is a fix or enhancement, I think it is worth to highlight. Reworded it slightly.

Reply:
distinct pushdown is a new feature

Reply:
It is listed under "features".